src/index.ts

```diff
@@ -3,6 +3,7 @@ export * from './Interceptor'
 export * from './BatchInterceptor'
 
 /* Utils */
+export { randomId } from './randomId'
```

Comment:
Export it so MSW can depend on the same ID generation. 
This doesn't really matter, just don't want to repeat the same function across projects.